makefile.unix

```diff
@@ -297,7 +299,7 @@ clean:
 install: $(LIBMAIN_S) $(HEADERS)
```

Comment:
maybe `install: $(LIBMAIN_S) $(HEADERS_PUB)`

Reply:
I removed it entirely, as it's already an indirect dependency of LIBMAIN_S